docs/dev/cli.md

```diff
@@ -17,7 +17,7 @@ hide_table_of_contents: true
 <div>
   <div className="box boxwidetop card">
     <div className="container">
-    <a href="/dev/cli/sauce-connect-5/run"><h3>Sauce Connect Proxy 5 CLI </h3></a>
+    <a href="/secure-connections/sauce-connect-5/cli/run"><h3>Sauce Connect Proxy 5 CLI </h3></a>
```

Comment:
Having cli with all the other docs has several advantages. As a user I would like to have everything at one place, that I can quickly find at the sidebar. At this moment SC5 docs and SC5 CLI come across as a separate documents. They lack connection IMO. 
Generally, I would like to achieve a sidebar similar to what we have here: https://forwarder-proxy.io.

The reference to SC5 CLI will stay at `/dev/cli/`, so this would satisfy our convention.